ansible/roles/ci/doc/README.md

```diff
@@ -0,0 +1,14 @@
+# Continuous Integration server support
+
+The main purpose of the CI server support is ability to use available hosted CI\CD server solutions for current and the future projects in agile fashion.
+In order to achieve this goal extra roles were developed for Ansible build tools.
+
+For more details please read following information
```

Comment:
Few suggestions:
"The main purpose of the CI server support is the ability to use available hosted CI\CD server solutions for current and future projects in an agile fashion."
"In order to achieve this goal, extra roles were developed for Ansible build tools."
"For more details please read following information:"